server/controllers/routes/teacherProfile.js

```diff
@@ -0,0 +1,11 @@
+const { getTeacherProfile } = require('../../database/quieres/teacherProfile');
+
+
+const teacherProfile = (req, res, next) => {
+  getTeacherProfile()
```

Comment:
this query currently selects all of the teachers in the database. If that's the goal, you should rename the API and the query to (for example) `getTeachersList`. 
If however the goal of this API is to retrieve one specific teacher profile (as the current name suggests), it needs to be parametrized with the ID of the profile to retrieve:
```
const { id } = req.params;
getTeacherProfile(id).then(...)
```